java/pom.xml

```diff
@@ -106,6 +106,20 @@
             </resource>
         </resources>
         <plugins>
+            <plugin>
```

Comment:
redefinition line 146

Reply:
handled already in https://github.com/4paradigm/OpenMLDB/blob/7edf33c5d6fad8883563d8b7d9f68d560fdceb89/java/pom.xml#L369-L380

Reply:
done